test/shopify_app/webhooks_manager_test.rb

```diff
@@ -34,10 +32,9 @@ class ShopifyApp::WebhooksManagerTest < ActiveSupport::TestCase
     end
   end
 
-  test "#create_webhooks when creating a webhook fails and the webhook exists, do not raise an error" do
+  test "#create_webhooks doesn't create webhooks that are already created" do
     webhook = stub(persisted?: false)
-    webhooks = all_webhook_topics.map{|t| stub(topic: t)}
-    ShopifyAPI::Webhook.stubs(create: webhook, all: webhooks)
+    ShopifyAPI::Webhook.stubs(create: webhook, all: all_mock_webhooks)
```

Comment:
this test was poorly named and the implementation was confusing. I re-wrote to better explain what we care about here